utils/uiTypes/proposalCreationTypes.ts

```diff
@@ -404,6 +410,14 @@ export interface UpdateTokenMetadataForm {
   programId: string | undefined
 }
 
+export interface UpdateTokenMetadataForm {
```

Comment:
This UpdateTokenMetadadaForm is just a random interface someone pushed that gives errors on compile.

I made no change to it.

Reply:
Ended up removing this piece of code as someone else resolved it.